view/adminhtml/templates/payment/script.phtml

```diff
@@ -18,7 +18,7 @@ $code = $block->escapeHtml($block->getCode());
             'jquery',
             'domReady!'
         ], function(Braintree, $) {
-            let config = <?= /* @noEscape */ $block->getPaymentConfig() ?>,
+            let config = <?= $block->escapeJs($block->getPaymentConfig()) ?>,
```

Comment:
escapeJs will escape the quotes - escapeJsQuote() should work here

Reply:
`escapeJsQuote` escapes quotes, however, it is marked as deprecated.

Reply:
In that case should any function be used here:

https://magento.stackexchange.com/questions/156368/magento-2-template-security-which-method-to-use
![image](https://user-images.githubusercontent.com/12492383/62277150-7296c900-b43d-11e9-8fe3-8a828efb8138.png)

As this is handling the output from getPaymentConfig() which returns a json_encode()?

Reply:
The escape method was mistakenly used as a code tidy pass was done. It was reverted back to being output without an escape method, as it was a JSON string being output. Crisis averted :)